src/components/gameboard/context-config.js

```diff
@@ -0,0 +1,88 @@
+import { createContext, useReducer } from 'react'
+import { createInitialLetterBag } from '../gameUtils'
+
+export const initalState = {
+  letterBag: createInitialLetterBag(),
+  drawnLetter: '',
+  playedLetter: '',
+  count: 0,
+  isTransitioning: false,
+  letterHand: [],
+}
+
+const actions = {
+  UPDATE_COUNT: 'UPDATE_COUNT',
+  SET_LETTERHAND: ' SET_LETTERHAND',
+  SET_LETTER_BAG: ' SET_LETTER_BAG',
+  SET_DRAWN_LETTER: ' SET_DRAWN_LETTER',
+  SET_LETTER_PLAYED: ' SET_LETTER_PLAYED',
+  IS_TRANSITIONING: 'SET_ISTRANSITIONING',
+}
+
+export const GameBoardContext = createContext(null)
+
```

Comment:
Thank you for implementing a reducer function. Please provide a short explanation of what you hoped to accomplish here.